x-pack/filebeat/modules.d/mysql.yml.disabled

```diff
@@ -0,0 +1,13 @@
+# Module: mysql
+# Docs: https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-module-mysql.html
+
+- module: mysql
```

Comment:
I'm pretty sure that this is going to cause issues. Might need a module name change:

https://github.com/elastic/beats/blob/f23f05a33fb570f7100a80d9ae31ab720ad96d1d/filebeat/module/mysql/_meta/config.yml#L1